rviz_default_plugins/CMakeLists.txt

```diff
@@ -585,7 +593,9 @@ if(BUILD_TESTING)
     ${SKIP_DISPLAY_TESTS})
   if(TARGET robot_test)
     target_include_directories(robot_test PUBLIC ${TEST_INCLUDE_DIRS})
-    target_link_libraries(robot_test ${TEST_LINK_LIBRARIES})
+    # TODO(clalancette): Figure out why putting resource_retriever in the
+    # TEST_TARGET_DEPENDENCIES doesn't work.
```

Comment:
This is likely an issue with resource_retriever's cmake no? It seems related to the changes @dirk-thomas has been making, but I don't have enough experience with those changes to say what might be the issue.

Reply:
Yeah, that very well could be.  I don't know why this isn't working here, and I also don't have enough experience.  These changes are enough to unblock the buildfarm (I think), but we should investigate why the transitive stuff doesn't seem to be working.